docs/docs/sample-scripts/spo/copy-listitems-sharepointlist.md

```diff
@@ -0,0 +1,32 @@
+# Copy list items between SharePoint lists
+
+Author: [SekThang](https://github.com/SekThang), Inspired by [Ruud](https://lazyadmin.nl/it/copy-sharepoint-list-items-to-another-list-with-powershell-and-pnp/)
+
+The cli script helps you to copy list items from one list to another list.
```

Comment:
Let's add instruction that user needs to update the script on line 22 and 24 with their columns.